config/locales/en.yml

```diff
@@ -44,6 +44,10 @@ en:
   statements_downcase: statements
   statemented_downcase: statemented
 
+  accounts_table:
```

Comment:
Not sure if this is the best place for this. Since this text is in JavaScript and the facility_accounts and user_accounts views, it wasn't super clear where else to put it